app/components/projects-discipline-selector.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { PropTypes } from 'react';
```

Comment:
Is this component even necessary, since it just returns an instance of `Filmstrip`?